samples/bluetooth/handsfree_ag/src/main.c

```diff
@@ -236,7 +236,7 @@ static void discovery_timeout_cb(const struct bt_br_discovery_result *results, s
 
 	for (i = 0; i < count; i++) {
 		bt_addr_to_str(&results[i].addr, addr, sizeof(addr));
-		printk("Device[%d]: %s, rssi %d, cod 0x%X%X%X", i, addr, results[i].rssi,
+		printk("Device[%d]: %s, rssi %d, cod 0x%02X%02X%02X", i, addr, results[i].rssi,
```

Comment:
Let's try to be consistent with lower case hex:
```
❯ git grep '0x%02x' subsys/bluetooth|wc -l
     533
❯ git grep '0x%02X' subsys/bluetooth|wc -l
     102
```

Reply:
OK, got it. Updated.